toolkit/scripts/pkggen.mk

```diff
@@ -80,31 +80,48 @@ $(graph_file): $(specs_file) $(go-grapher)
 		$(logging_command) \
 		--output $@
 
-ifeq ($(DISABLE_UPSTREAM_REPOS),y)
-graphpkgfetcher_disable_upstream_repos_flag := --disable-upstream-repos
-else
-graphpkgfetcher_disable_upstream_repos_flag :=
-endif
-
-ifeq ($(USE_UPDATE_REPO),y)
-graphpkgfetcher_update_repo_flag := --use-update-repo
-else
-graphpkgfetcher_update_repo_flag :=
-endif
-
-# We want to detect changes in the RPM cache, but we are not responsible for directly rebuilding any missing files.
-$(CACHED_RPMS_DIR)/%: ;
-
 # Remove any packages which don't need to be built, and flag any for rebuild if
```

Comment:
`graphoptimizer` no longer exists in `dev`. With exception of the changes to `graphpkgfetcher` in this file, the `dev` version should be taken wholesale.

Reply:
I've made the suggested changes - please double check